### ISSUE TITLE
v6 release?

### ISSUE THREAD

**Issue by mlankamp** [xBimTeam/XbimEssentials#526]
When is v6 going to be released? Or is there still work to be done?

**Comment by andyward** [comment]
Hi Michiel,

We're targeting January. V6 is basically done - there's a week or two of work just finishing up Geometry and flushing all the updates through from develop and the netcore feature branches across the toolkit - we just need to find some time in amongst other projects. If you wanted an early preview (it's pretty stable) check out the Myget packages https://www.myget.org/feed/xbim-develop/package/nuget/Xbim.Essentials

**Comment by mlankamp** [comment]
Good to hear!! Do you want to enable net8 (lts) also? It's due next week.

I can submit a PR if you like.

**Comment by andyward** [comment]
That would be great - thanks. We'll take a view on the Geometry and net8 - it's a bit more complex multi-targetting in the CLI/C++ side of things.

**Comment by mlankamp** [comment]
Just to be sure, PR against the development branch?

**Comment by andyward** [comment]
Yes PR against the `develop` branch please. We'll push to main/master

**Comment by mlankamp** [comment]
Created the PR: https://github.com/xBimTeam/XbimEssentials/pull/535

**Comment by simonedd** [comment]
Hi,

Is there any update on when v6 will be released? Thanks.

**Comment by martin1cerny** [comment]
Yes. We are just finalising our Flex Flow product for automated model processing (validation, transformations, enrichment) in the cloud. When we launch the trial, we will focus all our effort on the Toolkit v6. This should only be a few weeks away.

**Comment by ilCosmico** [comment]
I noticed that issue #544 has been fixed in v6. Could you please provide an ETA for this version?
